src/harness/testcases/testdata/fss_record_0.json

```diff
@@ -8,18 +8,21 @@
         "latitude": 39.2291,
         "longitude":  -100.1,
         "height": 7.6,
+        "elevation": 100,
```

Comment:
1. Note that we cannot use the `elevation` information, as it might be completely inconsistent with the terrain database elevation, and then it will break all propagation calculation. We shall use only the height above terrain and use the terrain from the database. Anyway the prop model do not have ability to make sense of such extra 'elevation'.
2. I though the FSS record is defined in the v1.0 of the SAS-SAS database and that we used that for defining the interface, even if it is no more defined in newer SAS-SAS database. 
On that doc, the FSS has been modeled with installationParams similar to the CBSD one. This is why we use antennaDowntilt, and not antennaElevationAngle.

I let @kate-harrison  comment further on that fact that we use the format defined for CBSD for the FSS record. So all non compliant info shall be removed, once this fact is well established among us.